github/event_types.go

```diff
@@ -149,6 +149,12 @@ type EditChange struct {
 	Title *struct {
 		From *string `json:"from,omitempty"`
 	} `json:"title,omitempty"`
+	Name *struct {
+		From *string `json:"from,omitempty"`
+	} `json:"title,omitempty"`
+	Note *struct {
+		From *string `json:"from,omitempty"`
+	} `json:"title,omitempty"`
 	Body *struct {
 		From *string `json:"from,omitempty"`
 	} `json:"body,omitempty"`
```

Comment:
I'm not sure that I fully understand the benefit of inlining the structs.

> That way, each changes field is going to contain exactly the fields that are supposed to be there, nothing more. I think it'd be better API.

Inlining the structs (by itself) does not solve this problem. There is no reason why a typed struct could not be created that contains exactly the fields that are supposed to be there with nothing more. It may mean that we have more typed structs, of course (which I think is OK).

As a user of the API, I would rather have an exported typed struct that I can fill in as opposed to having to copy/paste the full source code for the inlined struct if I need to use it.

I think we should err on the side of making the API easier to use for the end user, and inlining the structs seems to me that it makes it more difficult for the end user.